crud.py

```diff
@@ -0,0 +1,61 @@
+from sqlalchemy.orm import Session
+
+import models
```

Comment:
The import of `models` is redundant since `DBAuthor` and `DBBook` are already imported directly from `models`. Consider removing this import to clean up the code.